deploy/sre-prometheus/ocm-agent/obo-monitoring/100-oidc-missing.PrometheusRule.yaml

```diff
@@ -19,7 +19,8 @@ spec:
           # Clusters tend to have their `hypershift_cluster_invalid_aws_creds` set to > 0 while the HCP didn't finish the installation, thus we check
           # that the HCP is not rolling out in our expression (= hypershift_cluster_waiting_initial_avaibility_duration_seconds does not exist)
           # hypershift_cluster_waiting_initial_avaibility_duration_seconds stops being emitted once the HCP is rolled out
-          expr: sum by (mc_name, _mc_id, sector, region, env, namespace, exported_namespace, source, _id) (last_over_time(hypershift_cluster_invalid_aws_creds[10m])) > 0 unless on (exported_namespace) hypershift_cluster_waiting_initial_avaibility_duration_seconds unless on (exported_namespace) last_over_time(hypershift_cluster_deleting_duration_seconds[10m])
+          # This will be fixed with https://issues.redhat.com/browse/OCPBUGS-63353 
+          expr: (max by (mc_name, _mc_id, sector, region, env, namespace, exported_namespace, source, _id) (hypershift_cluster_invalid_aws_creds) == 1) unless on (exported_namespace) (hypershift_cluster_waiting_initial_avaibility_duration_seconds or hypershift_cluster_deleting_duration_seconds)
```

Comment:
All the possible values in the in the `max by` are all gonna be `1`s. So the `max by` will also result in a `1` (or nothing). More generally we do not care about the value returned by `max by`, so I would just replace it with `group by` which also return `1`.
This is of course a nice to have; but by using `group by`, you just definitely show that we do not care about the value.

Reply:
> All the possible values in the in the max by are all gonna be 1s

Is that right? I'm using `max by` to get a single value from two time series (two hypershift pods). I then care about whether or not that value is 0, 1 or 2.